buildpipeline/Core-Setup-CentOS-x64.json

```diff
@@ -3,6 +3,43 @@
     {
       "enabled": true,
       "continueOnError": false,
+      "alwaysRun": true,
```

Comment:
I'm not certain I understand the question.  `alwaysRun=true` implies that the step will run even if another step in the build fails.  `alwaysRun=false` means that if there is a build failure in any of the steps, then this task doesn't run.  I just assumed that this cleanup step was a latter step as we do it in the other repo's, but I see that it's not.  I'd actually move this to the end so that a failure in cleanup doesn't block the build from actually running.